portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,44 +39,84 @@
 public class DataServlet extends HttpServlet {
 
   private HashMap<String, HashMap<String,Object>> information;  
```

Comment:
1) Information is too generic of a name. Names should tell the reader what a variable holds; everything is information, so information can hold anything.
2) Why do we need to persist state in the Servlet itself and not use local variables in each method?